lib/client.js

```diff
@@ -510,7 +606,12 @@ MqttClient.prototype.subscribe = function () {
     var topics = []
     subs.forEach(function (sub) {
       if (that.options.reconnectPeriod > 0) {
-        that._resubscribeTopics[sub.topic] = sub.qos
+        that._resubscribeTopics[sub.topic] = { qos: sub.qos }
+        if (version === 5) {
+          that._resubscribeTopics[sub.topic].nl = sub.nl || false
+          that._resubscribeTopics[sub.topic].rap = sub.rap || false
+          that._resubscribeTopics[sub.topic].rh = sub.rh || 0
+        }
```

Comment:
we are manipulating a lot `_resubscribeTopics`. Can you please extract it into a separate variable?

Can you also add the docs for `nl` , `rap` and `rh`? If these are not exposed, can you please add a comment somewhere explaining them?

Reply:
fixed, added info in readme